hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainer.java

```diff
@@ -299,6 +299,9 @@ public void markContainerUnhealthy() throws StorageContainerException {
     } finally {
       writeUnlock();
     }
+    LOG.warn("Moving container {} to state UNHEALTHY from state:{}",
+            containerData.getContainerPath(), containerData.getState(),
+            new Exception());
```

Comment:
Is this ``new Exception()`` for adding the trace? Can we not use  ``StringUtils.getStackTrace(Thread.currentThread())``, else we would have an extra line saying ``java.lang.Exception`` without having any actual exception

Reply:
done